### DISCUSSION TITLE
Fantasy basketball matchup predictor

### DISCUSSION THREAD

**Discussion by ThomasAdriaanse** [cwendt94/espn-api#609] [Show and tell] [answered]
I have been doing a fun little side project to predict fantasy basketball matchups, feel free to check it out. This repo was a big help, thanks for all the work you have done!

As of now it is only for ESPN H2H points leagues, but I might add some add CAT stuff as well.

site:
[https://fantasybballstats.com](https://fantasybballstats.com/)

**Comment by cwendt94** [comment, marked as answer]
This is really cool, thanks for sharing!
